README.md

```diff
@@ -225,10 +226,153 @@ There are number of techniques you can use to ensure that access token expiry is
 - Retain the access token along with the timestamp of when it was requested so you can verify ahead of time whether the token has already expired based on the `time-to-live` being used by your server.
 - Prefetch the access token whenever the `UIApplication`, or `UIViewController` associated with an outgoing call is created.
 
+## New Features
+Voice iOS 3.X has a number of new features listed below:
+
+1. [WebRTC](#webrtc)
+2. [Custom Parameters](#custom-parameters)
+3. [Ringing](#ringing)
+4. [Stats](#media-stats)
+5. [TVODefaultAudioDevice](#tvodefaultaudiodevice)
+6. [TVOAudioDevice Protocol](#tvoaudiodevice-protocol)
+
+#### <a name="webrtc"></a>WebRTC
+The SDK is built using Chromium WebRTC for iOS. This ensures that over time developers will get the best real-time media streaming capabilities available for iOS. Additionally, upgrades to new versions of Chromium WebRTC will happen without changing the public API whenever possible.
+
+#### <a name="custom-parameters"></a>Custom Parameters
+Custom Parameters is now supported in `TVOCallInvite`. `TVOCallInvite.customParamaters` returns a `NSDictionary` of custom parameters sent from the caller side to the callee.
+
+Pass custom parameters in TwiML:
+
+```.xml
+<?xml version="1.0" encoding="UTF-8"?>
+<Response>
+  <Dial callerId="client:alice">
+    <Client>
+      <Identity>bob</Identity>
+      <Parameter name="caller_first_name" value="alice" />
+      <Parameter name="caller_last_name" value="smith" />
+    </Client>
+  </Dial>
+</Response>
+```
+
+`callInvite.customParameters` returns a dictionary of key-value pairs passed in the TwiML:
+
+```.objc
+{
+  "caller_first_name" = "alice";
+  "caller_last_name" = "smith";
+}
+```
+
+#### <a name="ringing"></a>Ringing
```

Comment:
Ringing APIs

Reply:
Followed Android PR: https://github.com/twilio/voice-quickstart-android/pull/212

Reply:
Happy to make these changes on the Android side. They are less terse headings.